tests/cross/src/test/scala/tests/pc/CompletionIssueSuite.scala

```diff
@@ -272,11 +272,11 @@ class CompletionIssueSuite extends BaseCompletionSuite {
     filter = (str) => str.contains("createProgress"),
     compat = Map(
       "3" -> """|import org.eclipse.lsp4j.services.LanguageClient
-                |import java.util.concurrent.CompletableFuture
                 |import org.eclipse.lsp4j.WorkDoneProgressCreateParams
+                |import java.{util => ju}
```

Comment:
Interesting, should we make it work the same for Scala 2 ?

Reply:
fixed https://github.com/scalameta/metals/pull/4037/commits/68892369439a232ba9e6874b0a5a8a70a19d2896
I realized Scala2 renames only if the direct owner symbol (of the symbol we're completing) is the `java.util`.